README.md

```diff
@@ -32,6 +32,7 @@ gulp.task('test', function (cb) {
       gulp.src(['test/*.js'])
         .pipe(mocha())
         .pipe(istanbul.writeReports()) // Creating the reports after tests runned
+        .pipe(istanbul.enforceThresholds()) // Check coverage against thresholds
```

Comment:
What do you mean by a configuration object? i.e. `{ thresholds: { global: 90 } }` ?
I'll improve the comment.